src/plugins/intel_cpu/src/ngraph_transformations/ref_convert_i64_i32.cpp

```diff
@@ -0,0 +1,36 @@
+// Copyright (C) 2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+
+
+#include "ref_convert_i64_i32.hpp"
+#include <openvino/opsets/opset10.hpp>
+#include "transformations/utils/utils.hpp"
+#include "cpu_types.h"
+
+
+bool ov::intel_cpu::RefConvertI64ToI32::run_on_model(const std::shared_ptr<ov::Model> &model) {
+    const auto orderedOps = model->get_ordered_ops();
+    for (const auto& op : orderedOps) {
+        if (TypeFromName(op->get_type_name()) == Type::Unknown) {
```

Comment:
Do I understand correctly that we insert convert only for extension nodes? Can we reflect this in the transformation description?

Reply:
Done